doc/boostbook/safefloat.xml

```diff
@@ -1239,43 +1239,157 @@ safe_float&lt;float, new_policy&gt; sf;
         
         <itemizedlist>
           <listitem>
-            policy::cast_from_primitive::none : Doesn't allow cast from any floating point primitive. Here and bellow "floating point primitive"
-            refers to types T such that <code>std::is_floating_point&lt;T&gt;::value == true</code>.
+            <para>
+              <code>cast_from_primitive::none</code>
+              : Doesn't allow cast from any floating point primitive. Here and bellow "floating point primitive"
+              refers to types T such that <code>std::is_floating_point&lt;T&gt;::value == true</code>.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>cast_from_primitive::same</code>
+              : Allow implicit cast from the wrapped floating point primitive.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>cast_from_primitive::same_explicit</code>
+              : Allow explicit cast from the wrapped floating point primitive.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>cast_from_primitive::more_precise</code>
+              : Allow implicit cast from floating point primitive with a precision
+              greater or equals to the precision of the wrapped type. The precision measured is
+              <code>std::numeric_limits&lt;FP&gt;::digits</code>
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>cast_from_primitive::more_precise_explicit</code>
+              : Allow explicit cast from floating point primitive with a precision
+              greater or equals to the precision of the wrapped type. The precision measured is
+              <code>std::numeric_limits&lt;FP&gt;::digits</code>
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>cast_from_primitive::less_precise</code>
+              : Allow implicit cast from floating point primitive with a precision
+              smaller or equals to the precision of the wrapped type. The precision measured is
```

Comment:
Why not bigger? It seems weird we restrict it to equall or less, and not just "anything".

Reply:
It is true that I can't think of a good reason to use it unlike more_precise but if the intend is to allow every floating point type, the option is still here with all